reconcile/sql_query.py

```diff
@@ -34,7 +34,7 @@
     spec:
       containers:
       - name: {{ JOB_NAME }}
-        image: quay.io/app-sre/{{ ENGINE }}:{{ENGINE_VERSION}}
+        image: quay.io/app-sre/qontract-reconcile-{{ENGINE}}:{{ENGINE_VERSION}}
```

Comment:
What about all the other versions?

Reply:
Isn't this swapped out for the `ENGINE_VERSION`?

Reply:
This would only support postgres 9.6  and mariadb 10.3 for now. We also have packages for postgres 10 and 12 which we can add as separate image tags. Looking at some of the previous uses of sql-query I have only seem 9.6 or 'latest' being used (default). For completeness we should also build postgresl 10 and 12

Reply:
If you mean postgres versions we will build the others as well

Reply:
We should build at least all versions on app interface. Or, maybe we can use latest for any version? Never tested it.

Reply:
versions are all built now with the latest changes

Reply:
Is the solution now ok?